### DISCUSSION TITLE
Hen house redesign

### DISCUSSION THREAD

**Discussion by LTNGx39** [sunflower-land/sunflower-land#2807] [Ideas]
# A facelift for hen houses
Over the course of a few months I have seen quite a few complaints or suggestions in the discord forum, commenting that the **hen houses are out of tune and are not aligned with the fences** in the game, asking that this be solved and if possible the fences of the building connect with the other fences.

> Thinking about this I decided to create a new design for the hen houses, without fences and also with a somewhat fresh touch but maintaining the original design

![Rediseños](https://github.com/sunflower-land/sunflower-land/assets/131133076/18106b44-8fde-4e19-9aad-a18abd07003e)

**Here you can see a comparison in which you can see what I commented and how the new design would look like in the game**

![Comparacion](https://github.com/sunflower-land/sunflower-land/assets/131133076/ef258a2f-41a2-48da-93d8-542215bad0fe)

It seems to me a simple solution to the problem and also an update that the community might like

# Resources

If you want to take my idea for the next updates, I have attached the necessary resources for its implementation. **Thank you for reading!**

| **GIF file** | ![hen_house](https://github.com/sunflower-land/sunflower-land/assets/131133076/707f5566-57f2-42f9-93ba-730d833f9a25) |
| - | - |
| Sprite sheet | ![hen_house_sprite_sheet](https://github.com/sunflower-land/sunflower-land/assets/131133076/c128ff63-d70d-4ede-b8ed-3a8590a7ab36) |

**Comment by TheDcol** [comment]
I am so for this 
![image](https://github.com/sunflower-land/sunflower-land/assets/108110065/582298eb-8115-4e02-8a3e-26de2a6b4bba)
